api/bin/chainflip-ingress-egress-tracker/src/witnessing/eth.rs

```diff
@@ -127,18 +127,11 @@ where
 			env_params.eth_vault_address,
 			Asset::Eth,
 			ForeignChain::Ethereum,
-			env_params.supported_erc20_tokens.clone(),
+			env_params.eth_supported_erc20_tokens.clone(),
 		)
 		.logging("witnessing Vault")
 		.spawn(scope);
 
-	let key_manager_address = state_chain_client
-		.storage_value::<pallet_cf_environment::EthereumKeyManagerAddress<state_chain_runtime::Runtime>>(
-			state_chain_client.latest_unfinalized_block().hash,
-		)
-		.await
-		.context("Failed to get KeyManager address from SC")?;
-
 	eth_source
 		.clone()
 		.key_manager_witnessing(witness_call.clone(), eth_client.clone(), key_manager_address)
```

Comment:
Dumb question, does this even compile? Since you are referencing the `key_manager_address` here, but removed the code above...

Reply:
Yeah just pushed the fix 👍  forgot to use the env params that I just put this in